### PR TITLE
[zh-cn]: Add translation for Text API

### PR DESCRIPTION
**Pull request by Saraph1nes**
Add translation for Text API

### CODE REVIEW

File: files/zh-cn/web/api/text/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Text
+slug: Web/API/Text
+---
+
+{{ApiRef("DOM")}}
+
+**`Text`** 接口表示 DOM 树中的一个文本节点（{{domxref("Node"， "Node")}}）。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/text/index.md:8]:
```suggestion
**`Text`** 接口表示 DOM 树中的一个文本{{domxref("Node", "节点")}}。
```

File: files/zh-cn/web/api/text/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: Text
+slug: Web/API/Text
+---
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/text/index.md:4]: add the `l10n.sourceCommit` key

File: files/zh-cn/web/api/text/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- `"\n  "` (在`</head>`结束标签之后，一个换行符后跟两个空格)
+- `"\n  "` (在`<body>`开始标签之后，一个换行符后跟两个空格)
+- `"\n Why yes.\n \n\n"` (`body`元素的内容)
+
+每个文本节点都是一个对象，具有本文中描述的属性和方法。
+
+## 构造函数
+
+- {{domxref("Text.Text", "Text()")}}
+  - : 返回一个新的`Text`节点，将参数作为其文本内容。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/text/index.md:38]:
```suggestion
  - : 返回一个新的 `Text` 节点，将参数作为其文本内容。
```

File: files/zh-cn/web/api/text/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+</html>
+```
+
+在这个文档中，有 5 个文本节点，其内容如下：
+
+- `"\n    "` (在`<head>`开始标签之后，一个换行符后跟四个空格)
+- `"Aliens?"` (`title`元素的内容)
+- `"\n  "` (在`</head>`结束标签之后，一个换行符后跟两个空格)
+- `"\n  "` (在`<body>`开始标签之后，一个换行符后跟两个空格)
+- `"\n Why yes.\n \n\n"` (`body`元素的内容)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/text/index.md:31]:
```suggestion
- `"\n    "`（在`<head>`开始标签之后，一个换行符后跟四个空格）
- `"Aliens?"`（`title` 元素的内容）
- `"\n  "`（在`</head>`结束标签之后，一个换行符后跟两个空格）
- `"\n  "`（在`<body>`开始标签之后，一个换行符后跟两个空格）
- `"\n Why yes.\n \n\n"`（`body` 元素的内容）
```

File: files/zh-cn/web/api/text/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+- {{domxref("Text.Text", "Text()")}}
+  - : 返回一个新的`Text`节点，将参数作为其文本内容。
+
+## 实例属性
+
+_从其父类继承属性，{{domxref("CharacterData")}}, {{domxref("Node")}}, 和 {{domxref("EventTarget")}}。_
+
+- {{domxref("Text.assignedSlot")}} {{ReadOnlyInline}}
+  - : 返回一个{{domxref("HTMLSlotElement")}}，表示当前节点所在的{{htmlelement("slot")}}。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/text/index.md:45]:
```suggestion
  - : 返回一个 {{domxref("HTMLSlotElement")}}，表示当前节点所在的 {{htmlelement("slot")}}。
```

File: files/zh-cn/web/api/text/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [MDN DOM 文档](/zh-CN/docs/Web/API/Document_Object_Model)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/text/index.md:66]:
```suggestion
- [DOM API](/zh-CN/docs/Web/API/Document_Object_Model)
```

**Review by skyclouds2001 — COMMENTED** [review]
source https://github.com/mdn/content/blob/main/files/en-us/web/api/text/index.md

File: files/zh-cn/web/api/text/index.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+---
+title: Text
+slug: Web/API/Text
+l10n:
+  sourceCommit: d414c502f3cc1c08d2fb043e98cda4a65621ff08
+---
+
+{{ApiRef("DOM")}}
+
+**`Text`** 接口表示 DOM 树中的一个文本{{domxref("Node", "节点")}}。
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:10]:
```suggestion
**`Text`** 接口表示 DOM 树中的一个文本{{domxref("Node", "节点", "", 1)}}。
```

File: files/zh-cn/web/api/text/index.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+每个文本节点都是一个对象，具有本文中描述的属性和方法。
+
+## 构造函数
+
+- {{domxref("Text.Text", "Text()")}}
+  - : 返回一个新的 `Text` 节点，将参数作为其文本内容。
+
+## 实例属性
+
+_从其父类继承属性，{{domxref("CharacterData")}}, {{domxref("Node")}}, 和 {{domxref("EventTarget")}}。_
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:44]:
```suggestion
_从其父类 {{domxref("CharacterData")}}、{{domxref("Node")}} 和 {{domxref("EventTarget")}} 继承属性。_
```

File: files/zh-cn/web/api/text/index.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+- {{domxref("Text.assignedSlot")}} {{ReadOnlyInline}}
+  - : 返回一个 {{domxref("HTMLSlotElement")}}，表示当前节点所在的{{htmlelement("slot")}}。
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:47]:
```suggestion
  - : 返回一个 {{domxref("HTMLSlotElement")}}，表示当前节点所在的 {{htmlelement("slot")}}。
```

File: files/zh-cn/web/api/text/index.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+- {{domxref("Text.wholeText")}} {{ReadOnlyInline}}
+  - : 获取相邻文本节点（{{domxref("Node")}}）的文本内容并拼接成一个字符串。
+
+## 实例方法
+
+_从其父类{{domxref("CharacterData")}}、{{domxref("Node")}}和{{domxref("EventTarget")}}继承方法。_
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:53]:
```suggestion
_从其父类 {{domxref("CharacterData")}}、{{domxref("Node")}} 和 {{domxref("EventTarget")}} 继承方法。_
```

File: files/zh-cn/web/api/text/index.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+  - : 返回一个新的 `Text` 节点，将参数作为其文本内容。
+
+## 实例属性
+
+_从其父类 {{domxref("CharacterData")}}、{{domxref("Node")}} 和 {{domxref("EventTarget")}} 继承属性。_
+
+- {{domxref("Text.assignedSlot")}} {{ReadOnlyInline}}
+  - : 返回一个 {{domxref("HTMLSlotElement")}}，表示当前节点所在的 {{htmlelement("slot")}}。
+- {{domxref("Text.wholeText")}} {{ReadOnlyInline}}
+  - : 获取相邻文本节点（{{domxref("Node")}}）的文本内容并拼接成一个字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:49]:
```suggestion
  - : 获取与当前{{domxref("Node", "节点", "", 1)}}相邻文本节点的文本内容并拼接成一个字符串。
```

File: files/zh-cn/web/api/text/index.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+  - : 返回一个新的 `Text` 节点，将参数作为其文本内容。
+
+## 实例属性
+
+_从其父类 {{domxref("CharacterData")}}、{{domxref("Node")}} 和 {{domxref("EventTarget")}} 继承属性。_
+
+- {{domxref("Text.assignedSlot")}} {{ReadOnlyInline}}
+  - : 返回一个 {{domxref("HTMLSlotElement")}}，表示当前节点所在的 {{htmlelement("slot")}}。
+- {{domxref("Text.wholeText")}} {{ReadOnlyInline}}
+  - : 获取与当前{{domxref("Node", "节点", "", 1)}}相邻文本节点的文本内容并拼接成一个字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/text/index.md:49]:
```suggestion
  - : 返回一个由与当前{{domxref("Node", "节点", "", 1)}}相邻的所有文本（`Text`）节点的文本内容按文档顺序拼接成一个字符串。
```

**Review by yin1999 — APPROVED** [review]
Thank you @Saraph1nes

**Pull request merged**